types/dialogflow/index.d.ts

```diff
@@ -1,6 +1,8 @@
-// Type definitions for dialogflow 0.9
+// Type definitions for dialogflow 1.0
```

Comment:
dialogflow package is currently at 0.9.1 not 1.0

Reply:
It's written on DT's readme to update to major version on breaking change. That's why I updated it to 1.0.